test/e2e/publicviewer/proxy_test.go

```diff
@@ -0,0 +1,181 @@
+package publicviewer_test
+
+import (
+	"context"
+	"testing"
+
+	"github.com/google/uuid"
+	"github.com/stretchr/testify/require"
+
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	waitpoll "k8s.io/apimachinery/pkg/util/wait"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+
+	toolchainv1alpha1 "github.com/codeready-toolchain/api/api/v1alpha1"
+	commonauth "github.com/codeready-toolchain/toolchain-common/pkg/test/auth"
+	. "github.com/codeready-toolchain/toolchain-e2e/testsupport"
+	authsupport "github.com/codeready-toolchain/toolchain-e2e/testsupport/auth"
+	"github.com/codeready-toolchain/toolchain-e2e/testsupport/spacebinding"
+	"github.com/codeready-toolchain/toolchain-e2e/testsupport/tiers"
+	"github.com/codeready-toolchain/toolchain-e2e/testsupport/wait"
+)
+
+type proxyUser struct {
+	expectedMemberCluster *wait.MemberAwaitility
+	username              string
+	token                 string
+	identityID            uuid.UUID
+	signup                *toolchainv1alpha1.UserSignup
+	compliantUsername     string
+}
+
+// tests access to community-shared spaces
+func TestPublicViewerProxy(t *testing.T) {
+	// given
+
+	// make sure everything is ready before running the actual tests
+	awaitilities := WaitForDeployments(t)
+	hostAwait := awaitilities.Host()
+	memberAwait := awaitilities.Member1()
+	// we create a space to share , a new MUR and a SpaceBindingRequest
+	space, _, _ := NewSpaceBindingRequest(t, awaitilities, memberAwait, hostAwait, "admin")
```

Comment:
why do you need to create the space using spacebinding request function ? It doesn't seem you use the SBR returned by this function, but maybe I'm missing something ? 

If you need only the Space you could use something like:
```
space, signup, _ := testsupportspace.CreateSpace(t, awaitilities, testspace.WithTierName("appstudio"), testspace.WithSpecTargetCluster(memberAwait.ClusterName))
```

Reply:
great catch!

Reply:
done in c4660e0332eb71ee3bec7645110ea5df8869ccb9